beater/checkintegration.go

```diff
@@ -42,7 +42,17 @@ func checkIntegrationInstalled(
 	kibanaClient kibana.Client,
 	esClient elasticsearch.Client,
 	logger *logp.Logger,
-) error {
+) (err error) {
+	defer func() {
+		if err != nil {
+			// We'd like to include some remediation actions when the APM Integration isn't installed.
+			err = &actionableError{
+				Err:         err,
+				Name:        "apm integration installed",
+				Remediation: "please install the apm integration",
```

Comment:
Generally +1 on introducing `actionableError`, but not certain how helpful this exact Remediation note is to users that are not familiar to the term `apm integration` or don't know how to install it. Ideally, if we are confident that doc links don't change frequently, we could link to the docs from here.

Reply:
We could add an ela.st shortened link (https://links.elastic.dev/), which can be edited. WDYT?

Reply:
That sounds perfect!

Reply:
Is this the correct URL? https://www.elastic.co/guide/en/apm/server/current/apm-integration.html